.github/workflows/release.yml

```
@@ -20,8 +20,8 @@ jobs:
       - name: Build distributions
         run: |
           python -m pip install --upgrade pip
-          python -m pip install build
-          python -m build
+          python -m pip install --upgrade build
+          python -m build --sdist --wheel
```

```suggestion
          python -m build
```
The default behavior with no arguments is more desirable here

> By default, a source distribution (sdist) is built from {srcdir} and a binary distribution (wheel) is built from the sdist. This is recommended as it will ensure the sdist can be used to build wheels. Pass -s/–sdist and/or -w/–wheel to build a specific distribution. If you do this, the default behavior will be disabled, and all artifacts will be built from {srcdir} (even if you combine -w/–wheel with -s/–sdist, the wheel will be built from {srcdir}).

https://build.pypa.io/en/stable/

---

Good catch! Updated to use `python -m build` with default behavior as suggested. This ensures the wheel is built from the sdist for better consistency.